reliability/core-it/src/androidTest/kotlin/com/datadog/android/core/integration/tests/BaseTest.kt

```diff
@@ -0,0 +1,18 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.core.integration.tests
+
+import java.util.concurrent.TimeUnit
+
+@Suppress("UtilityClassWithPublicConstructor")
+abstract class BaseTest {
+
+    companion object {
+        internal val LONG_WAIT_MS = TimeUnit.SECONDS.toMillis(30)
```

Comment:
I don't see this being used anywhere anymore. Can it be removed?

Reply:
I might use it later as some use - cases will require it (waiting for messages to be send through mock server). I would rather keep fit for now and decide later.